src/components/LeftMenu/index.tsx

```diff
@@ -0,0 +1,78 @@
+import DevName from "components/DevName";
+import DevNumber from "components/DevNumber";
+import DevLink from "components/DevLink";
+import * as S from "./styles";
+
+import { Heart as HeartIcon } from "@styled-icons/octicons/Heart";
+import { Star as StarIcon } from "@styled-icons/octicons/Star";
+import { People as PeopleIcon } from "@styled-icons/octicons/People";
+import { Organization as OrganizationIcon } from "@styled-icons/octicons/Organization";
+import { Location as LocationIcon } from "@styled-icons/octicons/Location";
+import { Mail as MailIcon } from "@styled-icons/octicons/Mail";
+import { Link as LinkIcon } from "@styled-icons/octicons/Link";
+
+import { Twitter as TwitterIcon } from "@styled-icons/feather/Twitter";
+import { Link } from "react-router-dom";
+import DevHeading from "components/DevHeading";
+
+export type UserProps = {
+  avatar: string;
+  devName: string;
+  devLogin: string;
+  devBio?: string;
+  followers: number;
+  following: number;
+  stars?: number;
+  organization?: string;
+  location?: string;
+  email?: string;
+  website?: string;
+  twitter?: string;
+};
+
+const LeftMenu = ({ ...props }: UserProps) => (
+  <S.Wrapper>
+    <S.Avatar src={props.avatar} />
+    <S.DevInfo>
+      <DevName>{props.devName}</DevName>
+      <DevName isLogin={true}>@{props.devLogin}</DevName>
+      {!!props.devBio && <S.DevBio>{props.devBio}</S.DevBio>}
+      <S.DevNumbers>
+        <DevNumber icon={<PeopleIcon />}>{props.followers} followers</DevNumber>
+        <DevNumber icon={<HeartIcon />}>{props.following} following</DevNumber>
+        <DevNumber icon={<StarIcon />}>{props.stars} stars</DevNumber>
+      </S.DevNumbers>
```

Comment:
Component is misaligned according to the layout.

![Captura de Tela 2021-04-24 às 01 22 45](https://user-images.githubusercontent.com/17678567/115947052-944ea400-a49b-11eb-9eb8-597bb5cdaf46.png)
![Captura de Tela 2021-04-24 às 01 23 02](https://user-images.githubusercontent.com/17678567/115947062-9e70a280-a49b-11eb-9a92-4ee8c2642dea.png)